CODEOWNERS

```diff
@@ -11,7 +11,7 @@
 /research/compression/ @nmjohn
 /research/deeplab/ @aquariusjay @yknzhu @gpapan
 /research/delf/ @andrefaraujo
-/research/differential_privacy/ @panyx0718 @mironov
+/research/differential_privacy/ @mironov @pseudorandom
```

Comment:
Is this the correct github uname? You appear to be @ilyamironov not mironov.

Reply:
And pseudorandom is probably not right either-- I suspect those are ldaps and not github usernames?

Reply:
My bad. Resubmitting.